Assets/HoloToolkit/Utilities/Scripts/SingleInstance.cs.meta

```diff
@@ -0,0 +1,12 @@
+fileFormatVersion: 2
+guid: 5ad8c5e16c3c3c045b63fba07276d0ab
```

Comment:
Is this the same meta file from the dev branch?

Reply:
Yes.
> fileFormatVersion: 2
> guid: 5ad8c5e16c3c3c045b63fba07276d0ab